client/data/marketplace/constants.ts

```diff
@@ -1,9 +1,101 @@
 export const BASE_STALE_TIME = 1000 * 60 * 60 * 2; // 2 hours
-export const DEFAULT_PAGE_SIZE = 24;
-export const DEFAULT_FIRST_PAGE = 1;
+export const DEFAULT_PAGE_SIZE = 20;
+export const DEFAULT_FIRST_PAGE = 0;
```

Comment:
What's the rationale here? Setting first page to 0 is not so intuitive 🤔

Reply:
This constant can be deleted as we are using `page_handle` instead, I will have a look